docs/record_contracts.md

```
@@ -799,18 +799,53 @@ cache の変更に入る前に、対応する現行種別名を示して停止
+- native `NL_UM`、標準名 `UMA`。どちらも同じ `KettoNum` 主キー・必須
+  header/key/body・公式 field capacity を使い、status 1/2/3/4/9 を provider 順に
+  同じ行へ反映します。
 - 現行の標準名 `UMA` では、公式レコード内の 27 組×6 着回数、4 脚質、登録
   レース数も個別列へ欠落なく展開し、抹消年月日の `00000000` は 8 文字のまま
   保持します。
+- UM は蓄積系（`DIFN`/`BLDN`）master だけであり、`RT_UM` は作成せず、速報経路
+  にもルーティングしません。
 
 **DataKubun:**
 
 - base domain は [公式 DataKubun の検証](#datakubun) の表と `UM=9` の
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)